bot/seasons/evergreen/recommend_game.py

```diff
@@ -0,0 +1,52 @@
+import json
+import logging
+import os
+from pathlib import Path
+from random import shuffle
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+DIR = "bot/resources/evergreen/game_recs"
+game_recs = []
+
+
+class RecommendGame(commands.Cog):
+    """Commands related to recommending games."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.populate_recs()
+
+    @commands.command(name="recommend_game")
```

Comment:
Any ideas for a clear name that doesn't have an underscore?

Reply:
gameRec maybe?

Reply:
I'll add `gameRec` and `recommendGame` as aliases for now.